### ISSUE TITLE
CUSTOM_AMI_NAME_PREFIX feature does not work with "pcluster createami"

### ISSUE THREAD

**Issue by rmarable** [aws/aws-parallelcluster#1550]
**Environment:**
 - AWS ParallelCluster / CfnCluster version [e.g. aws-parallelcluster-2.4.1] 2.5.0
 - OS: [e.g. alinux] OSX-10.14.6 (Mojave) with pyenv (Python-3.7.4)
 - Scheduler: [e.g. SGE] SGE
 - Master instance type: [e.g. m5.xlarge] r5.large
 - Compute instance type: [e.g. c5.8xlarge] r5.large

**Bug description and how to reproduce:**

The ParallelCluster createami" feature does not seem to support CUSTOM_AMI_NAME_PREFIX:
```
$ pcluster createami --ami-id [**************] --os centos7 -ap "ParallelClusterAmi-" -c pcluster-ami.config
Building AWS ParallelCluster AMI. This could take a while...

No custom AMI created
Unexpected error of type AttributeError: 'NoneType' object has no attribute 'get_param_value'
Traceback (most recent call last):
  File "/Users/rodneymarable/.pyenv/versions/3.7.4/envs/compbio-ami/lib/python3.7/site-packages/pcluster/cli.py", line 439, in main
    args.func(args)
  File "/Users/rodneymarable/.pyenv/versions/3.7.4/envs/compbio-ami/lib/python3.7/site-packages/pcluster/cli.py", line 81, in create_ami
    pcluster.create_ami(args)
  File "/Users/rodneymarable/.pyenv/versions/3.7.4/envs/compbio-ami/lib/python3.7/site-packages/pcluster/commands.py", line 831, in create_ami
    pcluster_config = PclusterConfig(config_file=args.config_file, fail_on_file_absence=True)
  File "/Users/rodneymarable/.pyenv/versions/3.7.4/envs/compbio-ami/lib/python3.7/site-packages/pcluster/config/pcluster_config.py", line 64, in __init__
    self._init_config_parser(config_file, fail_on_file_absence)
  File "/Users/rodneymarable/.pyenv/versions/3.7.4/envs/compbio-ami/lib/python3.7/site-packages/pcluster/config/pcluster_config.py", line 109, in _init_config_parser
    self.error(error_message)
  File "/Users/rodneymarable/.pyenv/versions/3.7.4/envs/compbio-ami/lib/python3.7/site-packages/pcluster/config/pcluster_config.py", line 446, in error
    if self.fail_on_error:
  File "/Users/rodneymarable/.pyenv/versions/3.7.4/envs/compbio-ami/lib/python3.7/site-packages/pcluster/config/pcluster_config.py", line 225, in fail_on_error
    self._fail_on_error = self.get_section("global").get_param_value("sanity_check")
AttributeError: 'NoneType' object has no attribute 'get_param_value'
```
More robust error handling to troubleshoot would also be appreciated: [https://github.com/aws/aws-parallelcluster/issues/1549](url)

**Comment by lukeseawalker** [comment]
Duplicate of https://github.com/aws/aws-parallelcluster/issues/1549

**Comment by lukeseawalker** [comment]
If you can post your config file we could see what's wrong in there and unblock you from running the command.

Thanks

**Comment by rmarable** [comment]
Hi Luca,

The config file I tried to use is attached.  Please let me know if you have any other questions.

[adam-pcluster.config.txt](https://github.com/aws/aws-parallelcluster/files/3952069/adam-pcluster.config.txt)

**Comment by lukeseawalker** [comment]
Try to remove the `-c` parameter and use the `--vpc-id` and `--subnet-id` instead.
